src/main/java/seedu/mealcompanion/command/recipe/RecipeFindCommand.java

```diff
@@ -0,0 +1,46 @@
+package seedu.mealcompanion.command.recipe;
+
+import seedu.mealcompanion.MealCompanionSession;
+import seedu.mealcompanion.command.ExecutableCommand;
+import seedu.mealcompanion.recipe.Recipe;
+import seedu.mealcompanion.recipe.RecipeList;
+
+//@@author jingyaaa
+
+public class RecipeFindCommand extends ExecutableCommand {
+
+    String findTerm;
+
+    public RecipeFindCommand(String findTerm) {
+        this.findTerm = findTerm;
+    }
+
+    private boolean findRecipes(MealCompanionSession mealCompanionSession) {
```

Comment:
Maybe it would be a good idea to use a more boolean sounding name for this method